ShareClientUI/ShareService+UI.swift

```diff
@@ -35,4 +35,6 @@ extension ShareService: ServiceAuthenticationUI {
             )
         ]
     }
```

Comment:
Used for NightscoutAPIClient but not needed here.

Reply:
I think we already have this change. Thanks!